### PR TITLE
Romanian translation added

### PR DESCRIPTION
**Pull request by vladutilie**
Hi @pschlan,

Finally (after more than four years 😁 - in 2019, I sent you an email about localizing cron-job.org into Romanian, and it was not possible at that time)!
The Romanian translation is here for: `frontend`, `statuspage`, and `api`.

Cheers,
Vlad

### CODE REVIEW

File: statuspage/src/utils/Config.default.js [hunk 1]
```diff
@@ -19,7 +19,8 @@ export const Config = {
     'en': 'English',
     'de': 'Deutsch',
     'it': 'Italiano',
-    'zh_TW': '���餤��'
+    'zh_TW': '���餤��',
```
Comment (pschlan) [inline statuspage/src/utils/Config.default.js:22]: Could you check why this got corrupted? (Double-check you're using UTF-8 when saving?)

**Comment by pschlan** [comment]
This is amazing, thanks so much! I added one comment which we should check before merging. Thanks again!

File: statuspage/src/utils/Config.default.js (hunk 1 above)
Comment (pschlan) [reply statuspage/src/utils/Config.default.js:22]: Actually this was already corrupt before. I fixed it in `master`. Could you rebase your change?

**Comment by vladutilie** [comment]
Oh my God! I have worked on master and discarded all my changes and lost them! 🤦‍♂️

**Comment by vladutilie** [comment]
I was able to roll back the changes. They can be merged now. 🙏🏻 #fridayGitDrama

**Pull request merged**